test/integration/inline_verifier_test.rb

```diff
@@ -0,0 +1,95 @@
+require "test_helper"
+
+class InlineVerifierTest < GhostferryTestCase
```

Comment:
There's no test that has to deal with multiple batches of copied rows, right? I think we should have that

Reply:
Do you mean multiple corrupted rows spread across multiple batches? The InlineVerifier will cause an error on the first corrupted row it detect tho.

Reply:
Hmm I see.. May still be worth enabling the trigger after a batch or two have been copied?

Reply:
I don't think it will be that different, also getting the timing right is still a bit difficult.